scripts/combine_clinical_encoded.R

```diff
@@ -0,0 +1,29 @@
+# Pan-Cancer Variational Autoencoder
+# Gregory Way 2017
+#
+# Combine VAE features with clinical data: for use in downstream analyses
+
+library(readr)
+library(dplyr)
+
+vae_file <- file.path("data", "encoded_rnaseq_onehidden_warmup_batchnorm.tsv")
+clinical_file <- file.path("data", "clinical_data.tsv")
+vae_df <- readr::read_tsv(vae_file)
+colnames(vae_df)[1] <- "sample_id"
+clinical_df <- readr::read_tsv(clinical_file)
+
+vae_df <- vae_df %>% dplyr::rowwise() %>% 
```

Comment:
Stylistically, I would put the function calls after the `%>%` consistent with the convention [here](http://magrittr.tidyverse.org/)

Reply:
:+1: